.ci/nightly-build-config.yaml

```diff
@@ -12,9 +12,6 @@ default:
   build-command:
     current: mvn clean install ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_CURRENT }}
     upstream: mvn clean install -Dquickly ${{ env.BUILD_MVN_OPTS }} ${{ env.BUILD_MVN_OPTS_UPSTREAM }}
-    after:
```

Comment:
this can be dropped as there is no need to invoke docker command in drools-ansible-integration nightly.